pages/transactional-email/how-to/manage-blocklists.mdx

```diff
@@ -0,0 +1,63 @@
+---
+meta:
+  title: How to manage Transactional Email blocklists
+  description: This page provides instructions how to improve the quality of your email deliveries and maintain your domain's reputation score with TEM blocklists.
+content:
+  h1: How to manage Transactional Email blocklists
+  paragraph: This page provides instructions how to improve the quality of your email deliveries and maintain your domain's reputation score with TEM blocklists.
+tags: blocklist reputation transactional
+dates:
+  validation: 2025-01-20
+  posted: 2025-01-20
+categories:
+  - managed-services
+---
+
+The Transactional Email blocklist feature allows your to manage email addresses that generate errors, or “bounces”. This helps you improve the quality of your email deliveries, maintain your domain's reputation score, and reduce the risk of having your domains blocked by ISPs.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- [Configured your domain(s) with Transactional Email](/managed-services/transactional-email/how-to/configure-domain-with-transactional-email/)
+
+## How to monitor temporary blocklists
+
+If any of your emails were bounced, you will see a temporary blocklist entry, with a duration of 48h, containing the details of the email that bounced and the reason, or flag. Some examples of bounce reasons are:
+
+  - `Mailbox not found` - when mailbox does not exist
+  - `Mailbox full` - when the mailbox is full.
+
+The temporary blocklist entry is created automatically to prevent repeated attempts to send e-mails, which could damage your reputation.
+
+To see the blocklist:
+
+1. Click **Transactional Email** in the **Managed Services** section of the [console](https://console.scaleway.com/) side menu. Your domains display.
+2. Click the domain for which you want to manage the blocklist. An overview of your domain appears.
+3. Click the **Blocklist** tab. Your blocklist displays detailing the recipient email, the flags, the date the block began and when it ends.
+
+<Message type="tip">
+  You can [create webhooks](/managed-services/transactional-email/how-to/create-webhooks) for the following blocklist events:
+    - `email_blocklisted` - An email was blocked by an active blocklist.
+    - `blocklist_created` - A blocklist is created for an email address. Any new emails sent by the address will be blocked.
+
+  By setting up a blocklist webhook, you can track changes to your blocklist in real time.
+</Message>
+
+## How to unblock an email address
```

Comment:
It could be useful to have a note mentioning the rate limit on this feature. A user cannot delete more than 5 blocklist during a 24 hours window.